packages/moonstone/ContextualPopupDecorator/ContextualPopupDecorator.js

```diff
@@ -495,6 +482,23 @@ const Decorator = hoc(defaultConfig, (config, Wrapped) => {
 			forward('onClose')
 		)
 
+		handleOpen = () => {
+			forward('onOpen', this.props);
```

Comment:
You've omitted the event payload here. It should likely be forwarded on from the param passed to `handleOpen`.

Reply:
:disappear: